src/FullBoard.cpp

```diff
@@ -132,7 +131,7 @@ int FullBoard::update_board(const int color, const int i) {
     auto eyeplay = (m_neighbours[i] & s_eyemask[!color]);
 
     auto captured_stones = 0;
-    int captured_sq;
+    auto captured_sq = -1;
```

Comment:
This is not required. (Despite what the stupid compiler says).

Reply:
I know it's not required but I'm a big fan of reducing warnings so that when they do point to real bugs people pay attention to them.

Reply:
We can disable that compiler warning. It's even generating warnings in boost code for me, to clearly it's a bit too spammy.